### ISSUE TITLE
[py2-v2] healpix-cxx not correctly linked against cfitsio (Trac #1344)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1336]
Link error:

```text
/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/lib64/libhealpix_cxx.so: undefined reference to `fitshandle::read_column_raw_void(int, void*, PDT, long, long) const'
/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/lib64/libhealpix_cxx.so: undefined reference to `fitshandle::efficientChunkSize(int) const'
collect2: ld returned 1 exit status
make[3]: *** [bin/NoiseEngine-test] Error 1
make[2]: *** [NoiseEngine/CMakeFiles/NoiseEngine-test.dir/all] Error 2
make[1]: *** [NoiseEngine/CMakeFiles/NoiseEngine-test.dir/rule] Error 2
make: *** [NoiseEngine-test] Error 2
```

Migrated from https://code.icecube.wisc.edu/ticket/1344
```json
{
    "status": "closed", 
    "changetime": "2015-09-22T22:46:49", 
    "description": "Link error:\n{{{\n/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/lib64/libhealpix_cxx.so: undefined reference to `fitshandle::read_column_raw_void(int, void*, PDT, long, long) const'\n/cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/lib64/libhealpix_cxx.so: undefined reference to `fitshandle::efficientChunkSize(int) const'\ncollect2: ld returned 1 exit status\nmake[3]: *** [bin/NoiseEngine-test] Error 1\nmake[2]: *** [NoiseEngine/CMakeFiles/NoiseEngine-test.dir/all] Error 2\nmake[1]: *** [NoiseEngine/CMakeFiles/NoiseEngine-test.dir/rule] Error 2\nmake: *** [NoiseEngine-test] Error 2\n}}}", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1442962009995392", 
    "component": "cvmfs", 
    "summary": "[py2-v2] healpix-cxx not correctly linked against cfitsio", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-09-14T19:58:05", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150915T07:25:18: 
* david.schultz changed status from "new" to "accepted"
* david.schultz changed summary from "[healpix-cxx] not correctly linked against cfitsio in py2-v2" to "[py2-v2] healpix-cxx not correctly linked against cfitsio"
* david.schultz changed description from:

> [https://icecube-spno.slack.com/files/kkrings/F0AL90XJQ/noiseengine-test_compile_error_with_py2-v2.txt]

to:

> Link error:
> {{{
> /cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/lib64/libhealpix_cxx.so: undefined reference to `fitshandle::read_column_raw_void(int, void*, PDT, long, long) const'
> /cvmfs/icecube.opensciencegrid.org/py2-v2/RHEL_6_x86_64/lib64/libhealpix_cxx.so: undefined reference to `fitshandle::efficientChunkSize(int) const'
> collect2: ld returned 1 exit status
> make[3]: *** [bin/NoiseEngine-test] Error 1
> make[2]: *** [NoiseEngine/CMakeFiles/NoiseEngine-test.dir/all] Error 2
> make[1]: *** [NoiseEngine/CMakeFiles/NoiseEngine-test.dir/rule] Error 2
> make: *** [NoiseEngine-test] Error 2
> }}}

**Comment by icecube-trac** [comment]
Trac update at 20150915T08:00:29: 
* david.schultz changed status from "accepted" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 137577:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137577"
> fix healpix_cxx linking. closes #1336
> ```

**Comment by icecube-trac** [comment]
Trac update at 20150915T08:38:59: 
* david.schultz changed resolution from "fixed" to ""
* david.schultz changed status from "closed" to "reopened"
* david.schultz commented:

> apparently that didn't do it.  must be more library linking issues (maybe the sub-libraries aren't getting linked in?)

**Comment by icecube-trac** [comment]
Trac update at 20150922T22:46:49: 
* david.schultz commented:

> In changeset 137855:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137855"
> really fix #1336 this time
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "reopened" to "closed"
